### DISCUSSION TITLE
clear seach characters after each jump

### DISCUSSION THREAD

**Discussion by rhythm16** [folke/flash.nvim#129] [Q&A] [answered]
hi, 

Thank you for this awesome plugin!

Currently I have to backspace when ever I initiate a new jump, for example first I want to look for "abc", I hit s, then type abc.
I see ⚡abc at the lower left corner of the editor and all the jump candidates, then I hit a key to go to a jump label.
After a few operations I want to do another jump and look for another sequence of characters, lets say "xyz", as I hit s again, ⚡abc shows up again at the corner with all the abc candidates highlighted.
And I would have to backspace and clear "abc", then type xyz to see the jump candidates for xyz.
Is it possible to clear the search every time a new jump is initiated (just like we have to retype what we want to search everytime for  '/')?
I have tried a few config options but none seems to do what I wished.
Thank you!

**Comment by folke** [comment]
That's not how flash should work. Did you add continue=true?

**Reply by rhythm16** [reply]
I am using
```lua
-- flash.lua
local status_ok, flash = pcall(require, "flash")    
if not status_ok then    
    return    
end    

flash.setup {
    continue = false,
}
```
and 
```lua
-- keymaps.lua
local opts = { noremap = true, silent = true }
local keymap = vim.api.nvim_set_keymap
keymap("n", "s", ":lua require(\"flash\").jump({continue=false})<CR>", opts)
```
to map s to use flash jump
however, whether continue=true or continue=false does not change the said behavior

here is a demonstration of my situation, I would like to not have to backspace and clear "key" before I can search for "map".

[Screencast from 2023年七月06日 11時38分41秒.webm](https://github.com/folke/flash.nvim/assets/31589689/bb6910da-064f-467b-8d8e-1713085f1f20)

**Comment by folke** [comment, marked as answer]
just use the mappings from the docs. no need to add that lua string. you can just use a function

**Reply by rhythm16** [reply]
I changed the keymaps and it worked!
Thanks :)
